src/medCore/database/medDatabaseExporter.cpp

```diff
@@ -59,7 +73,18 @@ void medDatabaseExporter::internalRun(void)
     }
 
     dtkAbstractDataWriter * dataWriter = medAbstractDataFactory::instance()->writer(d->writer);
-    dataWriter->setData(d->data);
+    if(!d->saveMultipleData)
+    {
+        dataWriter->setData(d->data);
+    }
+    else
+    {
+        medAbstractDataWriter* medDataWriter = dynamic_cast<medAbstractDataWriter*>(dataWriter);
+        if(medDataWriter)
```

Comment:
a small comment: why do you need to test `medDataWriter` here, and not line 78 with `dataWriter`? If you want, you can test `dataWriter` after line 61, and no need to test again medDataWriter